.github/workflows/test_generator_backend.yml

```
@@ -0,0 +1,98 @@
+name: Test Generator Backend
+
+# Exécution en priorité à chaque push ou PR sur main/develop
+on:
+  push:
+    branches: [main, develop]
+    paths:
+      - 'backend/**'
+      - 'tools/test_generator/backend/**'
+    paths-ignore:
```

actionlint (1.7.4)

both "paths" and "paths-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)